controller/workitemtype_blackbox_test.go

```diff
@@ -98,13 +101,22 @@ func (s *workItemTypeSuite) SetupTest() {
 	require.NotNil(s.T(), s.spaceCtrl)
 }
 
+func (s *workItemTypeSuite) TearDownTest() {
+	s.clean()
+}
```

Comment:
This pattern is duplicated a lot, too. I wonder if we could factorize it once for all...